### ISSUE TITLE
xp

### ISSUE THREAD

**Issue by ddjunho** [ddjunho/upbitauto#2]
import pandas as pd
import numpy as np
import pyupbit
import tensorflow as tf
from sklearn.preprocessing import MinMaxScaler

# 티커 설정
ticker = "KRW-BTC"

# 데이터 불러오기
df = pyupbit.get_ohlcv(ticker, interval="minute15", count=192)

# 입력 데이터 전처리
X = df[['high', 'low', 'volume']].values # 입력 데이터는 high, low, volume 3가지 종류
X_scaler = MinMaxScaler()
X = X_scaler.fit_transform(X)

# 출력 데이터 전처리
y = df['high'].values # 출력 데이터는 high 가격
y_scaler = MinMaxScaler()
y = y_scaler.fit_transform(y.reshape((-1, 1)))

# 학습 데이터 생성
X_train = []
y_train = []
for i in range(, len(X)):
    X_train.append(X[i-192:i, :])
    y_train.append(y[i, 0])

X_train = np.array(X_train)
y_train = np.array(y_train)

# Tensorflow 모델 구성
model = tf.keras.models.Sequential([
    tf.keras.layers.LSTM(128, input_shape=(X_train.shape[1], X_train.shape[2])),
    tf.keras.layers.Dense(64, activation='relu'),
    tf.keras.layers.Dense(1)
])

# 모델 컴파일
model.compile(optimizer='adam', loss='mse')

# 학습
model.fit(X_train, y_train, epochs=100, verbose=0)

# 새로운 데이터에 대한 예측
last_data = df[['high', 'low', 'volume']].iloc[-192:].values # 가장 최근 192개 데이터
last_data = X_scaler.transform(last_data.reshape((1, -1, 3))) # 입력 데이터 전처리

predicted_price = model.predict(last_data) # 예측 결과
predicted_price = y_scaler.inverse_transform(predicted_price) # 출력 데이터 역전처리

# 예측 결과 출력
print("Predicted highest price:", predicted_price)

이 코드에서 192로 학습하게되 있지? 문제나 오류는 있나?
